sources/packages/web/src/components/aest/students/bypassedRestrictions/HistoryBypassedRestrictions.vue

```diff
@@ -67,17 +110,25 @@ import {
   ITEMS_PER_PAGE,
   ApplicationRestrictionManagementHeaders,
   RestrictionStatus,
+  Role,
 } from "@/types";
 import { ref, onMounted, defineComponent } from "vue";
 import StatusChipBypass from "@/components/generic/StatusChipBypass.vue";
 import { ApplicationRestrictionBypassService } from "@/services/ApplicationRestrictionBypassService";
 import { ApplicationRestrictionBypassHistoryAPIOutDTO } from "@/services/http/dto";
 import StatusChipRestriction from "@/components/generic/StatusChipRestriction.vue";
+import CheckPermissionRole from "@/components/generic/CheckPermissionRole.vue";
+import { ModalDialog } from "@/composables";
+import BypassRestrictionModal from "@/components/aest/students/modals/BypassRestrictionModal.vue";
+import RemoveRestrictionBypassModal from "@/components/aest/students/modals/RemoveRestrictionBypassModal.vue";
```

Comment:
As as suggestion, these modals can be added to the same folder since they are coupled.